mtp_api/apps/mtp_auth/tests/test_views.py

```diff
@@ -110,6 +110,16 @@ def test_user_id_added_to_job_information_record(self):
         self.assertEqual(len(records), 1)
         self.assertEqual(records[0].user_id, self.security_staff.id)
 
+    def test_flag_created_when_job_info_saved(self):
+        self.client.post(
+            self.url,
+            data=self.payload,
+            HTTP_AUTHORIZATION=self.get_http_authorization_for_user(self.security_staff)
+        )
+
+        flag_exists = Flag.objects.filter(user_id=self.security_staff.id, name='provided-job-information').exists()
```

Comment:
not sure i understand the second question in the PR: what this is doing makes sense to me.

could be a tiny bit slightly simpler with `…user=self.security_staff…`